### PR TITLE
Added environment objects to planning scene.

### PR DESCRIPTION
**Pull request by MaxGroenenboom**
Added some environment objects to the MoveIt! planning scene:
- The table the arm stands on.
- A backpanel to prevent the arm from using the space behind it (there would be other plants there).
- A temporary box representing the gripper.

Also added the option to enable the MoveIt! planning interface for live runs and to make it automatically enabled for testing runs. This interface is needed to view the added planning scene collision objects.

A first version of point cloud usage by the collision model has been added as well, but this has not been tested yet, and it doesn't have real parameters.

### CODE REVIEW

File: catkin_ws/src/universal_robot/ur5_moveit_config/launch/ur5_moveit_sensor_manager.launch.xml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 <launch>
-
+	<rosparam command="load" file="$(find ur5_moveit_config)/config/sensors_bumblebee.yaml" />
+	<param name="octomap_frame" type="string" value="odom_combined" />
+	<param name="octomap_resolution" type="double" value="0.05" />
+	<param name="max_range" type="double" value="2.0" />
 </launch>
```
Comment (ChielBruin) [inline catkin_ws/src/universal_robot/ur5_moveit_config/launch/ur5_moveit_sensor_manager.launch.xml:6]: Why is this a .launch.xml file and not simply .launch?

File: catkin_ws/src/universal_robot/ur5_moveit_config/config/sensors_bumblebee.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+sensors:
+  - sensor_plugin: occupancy_map_monitor/PointCloudOctomapUpdater
+    point_cloud_topic: points
+    #//TODO enter correct values
+    max_range: 2.0
+    point_subsample: 1
+    padding_offset: 0.1
+    padding_scale: 0.1
+    filtered_cloud_topic: filtered_cloud
```
Comment (ChielBruin) [inline catkin_ws/src/universal_robot/ur5_moveit_config/config/sensors_bumblebee.yaml]: Please test this

**Review by ChielBruin — CHANGES_REQUESTED** [review]
The code looks good, please address my comments and make it mergable again. Than we have to wait until travis has fixed itself

File: catkin_ws/src/universal_robot/ur5_moveit_config/launch/ur5_moveit_sensor_manager.launch.xml (hunk 1 above)
Comment (MaxGroenenboom) [reply catkin_ws/src/universal_robot/ur5_moveit_config/launch/ur5_moveit_sensor_manager.launch.xml:6]: It's a default file in that project, I merely added some parameters to it ;)

File: catkin_ws/src/universal_robot/ur5_moveit_config/config/sensors_bumblebee.yaml (hunk 2 above)
Comment (MaxGroenenboom) [reply catkin_ws/src/universal_robot/ur5_moveit_config/config/sensors_bumblebee.yaml]: I've tried testing it but currently `/filtered_cloud` doesn't return anything.

Comment (ChielBruin) [reply catkin_ws/src/universal_robot/ur5_moveit_config/config/sensors_bumblebee.yaml]: Than it should be fixed before merging

**Review by ChielBruin — APPROVED** [review]

**Pull request merged**